transport/http/handler.go

```diff
@@ -144,8 +143,12 @@ func handleOnewayRequest(
 	}
 	treq.Body = &buff
 
+	// create a new context for oneway requests since the HTTP handler cancels
+	// http.Request's context when ServeHTTP returns
+	ctx := opentracing.ContextWithSpan(context.Background(), span)
```

Comment:
The HTTP Oneway should be able to tell, to the client, whether the message was delivered using an Ack. 

Once it does that, it shouldn't wait to find out if the handler processed the message correctly or not.